src/main/java/org/openstreetmap/atlas/geography/atlas/change/MemberMergeStrategies.java

```diff
@@ -226,10 +221,12 @@
                 resultBean.addItem(resultKey);
             }
         }
-        /*
-         * TODO note we currently do not properly merge the explicitlyExcluded sets. The plan is to
-         * remove these sets. If we cannot find a way to do so, then we must merge them here.
-         */
+
+        afterLeftBean.getExplicitlyExcluded().stream()
```

Comment:
You can also concat these streams and then add
```
Stream.concat(afterLeftBean.getExplicitlyExcluded().stream(), afterRightBean.getExplicitlyExcluded().stream()).forEach(resultBean::addItemExplicitlyExcluded)
```

Reply:
Oh this is neat. Did not know about this.